storybook/stories/API/cartesian/Area.stories.tsx

```diff
@@ -84,10 +182,121 @@ const renderLabel = (props: { index: number; x: number; y: number }) => {
+    const { data, dataKey1, dataKey2, ...areaArgs } = args;
+
+    return (
+      <ResponsiveContainer width="100%" height={surfaceHeight}>
+        <ComposedChart
+          width={surfaceWidth}
+          height={surfaceHeight}
+          margin={{
+            top: 20,
+            right: 20,
+            bottom: 20,
+            left: 20,
+          }}
+          data={data}
+        >
+          <defs>
+            <linearGradient id="colorUv" x1="0" y1="0" x2="0" y2="1">
+              <stop offset="5%" stopColor="#8884d8" stopOpacity={0.8} />
+              <stop offset="95%" stopColor="#8884d8" stopOpacity={0} />
+            </linearGradient>
+            <linearGradient id="colorPv" x1="0" y1="0" x2="0" y2="1">
+              <stop offset="5%" stopColor="#82ca9d" stopOpacity={0.8} />
+              <stop offset="95%" stopColor="#82ca9d" stopOpacity={0} />
+            </linearGradient>
+          </defs>
+          <Area
+            type="monotone"
+            dataKey={dataKey1}
+            stroke="#8884d8"
+            fillOpacity={1}
+            fill="url(#colorUv)"
+            {...areaArgs}
+          />
+          <Area
+            type="monotone"
+            dataKey={dataKey2}
+            stroke="#82ca9d"
+            fillOpacity={1}
+            fill="url(#colorPv)"
+            {...areaArgs}
+          />
+        </ComposedChart>
+      </ResponsiveContainer>
+    );
+  },
+  args: {
+    data: pageData,
+    dataKey1: 'uv',
+    dataKey2: 'pv',
+    isAnimationActive: false,
+  },
+  parameters: {
+    controls: {
+      include: ['data'],
+    },
+  },
+};
+
+export const Animation = {
+  ...Basic,
+  args: {
+    data: coordinateWithValueData,
+    isAnimationActive: true,
+    animationEasing: 'linear',
+    animationBegin: 0,
+    animationDuration: 1500,
+  },
+  parameters: {
+    controls: { include: ['animationEasing', 'isAnimationActive', 'animationBegin', 'animationDuration'] },
+  },
+};
+
+export const Points = {
+  render: (args: Record<string, any>) => {
+    const { points } = args;
+
+    return (
+      <ResponsiveContainer width="100%" height={surfaceHeight}>
+        <Surface
+          width={surfaceWidth}
+          height={surfaceHeight}
+          viewBox={{
+            x: 0,
+            y: 0,
+            width: surfaceWidth,
+            height: surfaceHeight,
+          }}
+        >
+          <Area dataKey="" isAnimationActive={false} points={points} />
+        </Surface>
+      </ResponsiveContainer>
+    );
+  },
+  args: {
+    points: coordinateWithValueData,
+  },
+  parameters: {
+    controls: { include: ['points'] },
+    docs: {
+      description: {
+        story:
+          'You can directly set the x and y coordinates of a Area via points. This overrides dataKey and data. ' +
+          'The coordinate system of the points lies in the top right of the bounding box. ' +
+          'Using points, a Area can even be used within only a Surface, without a Chart.',
```

Comment:
<img width="1039" alt="Screen Shot 2023-02-24 at 6 40 35 AM" src="https://user-images.githubusercontent.com/60944077/221181563-25b1372b-df2b-45a8-8c19-230751d40d46.png">

Using back-ticks we can make it a little more clear that we're referring specific props in the code here.

```suggestion
          'You can directly set the x and y coordinates of a Area via `points`. This overrides `dataKey` and `data`. ' +
          'The coordinate system of the `points` lies in the top right of the bounding box. ' +
          'Using `points`, an Area can even be used within only a Surface, without a Chart.',
```

Reply:
Going to commit this suggestion and merge